src/endpoints/payment-method/payment-method.controller.ts

```diff
@@ -0,0 +1,31 @@
+import {Controller, Get, Post, Body, Put, Param, Delete} from '@nestjs/common';
+import {ApiParam} from '@nestjs/swagger';
+import {PaymentMethod} from 'src/models';
+import { PaymentMethodService } from './payment-method.service';
+
+@Controller('payment_method')
```

Comment:
@RamaMahjoub 
Could you please change the endpoint name to `payment-method` ?

See https://restfulapi.net/resource-naming/   (2.2.3. Use hyphens (-) to improve the readability of URIs)